tests/alpaka/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+traccc_add_test( alpaka
+   alpaka_main.cpp
+   LINK_LIBRARIES
+   GTest::gtest
+   alpaka
```

Comment:
The `alpaka` target (I believe) does work in all circumstances, but just to show a bit more clearly that this is something "external", I would prefer to use the `alpaka::alpaka` target instead.

https://github.com/alpaka-group/alpaka/blob/develop/cmake/alpakaCommon.cmake#L154